modules/develop/pages/config-topics.adoc

```diff
@@ -82,18 +82,44 @@ The cleanup policy determines how to clean up the partition log files when they
 * `compact` compacts the data by only keeping the latest values for each KEY. For details on compaction in Redpanda, see xref:manage:cluster-maintenance/compaction-settings.adoc[Compaction settings].
 * `compact,delete` combines both methods.
 
-Unlike compacted topics which keep only the most recent message for a given key, topics configured with a delete cleanup policy provide a running history of all changes for those topics. 
+Unlike compacted topics, which keep only the most recent message for a given key, topics configured with a `delete` cleanup policy provide a running history of all changes for those topics. 
 
-Suppose you configure a topic using the default cleanup policy `delete`, and you want to change the policy to `compact`. Run the `rpk topic alter-config` command as shown:
+For example, to change a topic's policy to `compact`, run:
 
 [,bash]
 ----
 rpk topic alter-config [TOPICS…] —-set cleanup.policy=compact
 ----
 
+=== Configure write caching
+
+Write caching is a relaxed mode of xref:develop:produce-data/configure-producers.adoc#acksall[`acks=all`] that provides better performance at the expense of durability. It acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to fsync to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write. 
+
+Write caching applies to user topics. It does not work with transactions or consumer offsets:  transactions and offset commits are always fsynced. 
+
+For clusters in xref:reference:rpk/rpk-redpanda/rpk-redpanda-mode.adoc#development-mode[development mode], write caching is enabled by default. For clusters in production mode, it is disabled. 
+
+NOTE: Only enable write caching on workloads that can tolerate some data loss in the case of multiple simultaneous broker failures. Leaving write caching disabled safeguards your data against complete data center or availability zone failures. 
+
+==== Configure at cluster level
+
+To enable write caching in production mode, set the cluster-level property xref:reference:cluster-properties.adoc#write_caching[`write_caching`]:
+
+`rpk cluster config set write_caching=on`
+
+With `write_caching` enabled at the cluster level, Redpanda fsyncs to disk according to `raft_replica_max_pending_flush_bytes` and `raft_replica_max_flush_delay_ms`, whichever is reached first.
```

Comment:
^^ Please add these two properties to the property reference and then link to those entries from here. Thx.